frontend/src/utils/formatCurrencyUtil.ts

```diff
@@ -0,0 +1,10 @@
+export const formatCurrency = (number: number | null) => {
+  if (number) {
```

Comment:
can we name this variable something other than `number` to avoid a collision with the primitive type?